packages/EDA/src/package.ts

```diff
@@ -734,3 +738,55 @@ export async function visualizePLSRegression(df: DG.DataFrame, targetColumn: DG.
 export function isInteractivePLSRegression(df: DG.DataFrame, predictColumn: DG.Column): boolean {
   return PlsModel.isInteractive(df.columns, predictColumn);
 }
+
+//name: trainXGBooster
+//meta.mlname: XGBoost
+//meta.mlrole: train
+//input: dataframe df
+//input: column predictColumn
+//input: int iterations = 20 {min: 1; max: 100} [Number of training iterations]
+//input: double eta = 0.3 {caption: Rate; min: 0; max: 1} [Learning rate]
+//input: int maxDepth = 6 {min: 0; max: 20} [Maximum depth of a tree]
+//input: double lambda = 1 {min: 0; max: 100} [L2 regularization term]
+//input: double alpha = 0 {min: 0; max: 100} [L1 regularization term]
+//output: dynamic model
+export async function trainXGBooster(df: DG.DataFrame, predictColumn: DG.Column,
```

Comment:
Is there a reason to call it XGBooster instead of just XGBoost? Second seems more natural and common for me